app/models/shelter.rb

```diff
@@ -1,3 +1,4 @@
 class Shelter < ApplicationRecord
   has_many :pets, dependent: :destroy
+  has_many :reviews, dependent: :destroy
```

Comment:
do we need to add to the shelter model test?

Reply:
Yep, missed this one as well. I will update. Thanks!